charts/templates/configmap.yaml

```diff
@@ -23,8 +23,8 @@ data:
   MARKLOGIC_JOIN_TLS_ENABLED: "false"
 {{- end }}
   MARKLOGIC_FQDN_SUFFIX: {{ include "marklogic.headlessURL" . }}
-  MARKLOGIC_INIT: "true"
-  MARKLOGIC_JOIN_CLUSTER: "true"
+  MARKLOGIC_INIT: "false"
+  MARKLOGIC_JOIN_CLUSTER: "false"
```

Comment:
Why do we set both values to be false?

Reply:
As we discussed, for Helm 2.0, we set both MARKLOGIC_INIT and MARKLOGIC_JOIN_CLUSTER to false.